docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst

```diff
@@ -13,8 +13,92 @@ Case definition in GBD 2017
 Neonatal sepsis and other neonatal infections are infections during the neonatal
 period that advance to a systemic bloodstream infection, the underlying cause of
 which can be meningitis, gastroenteritis, or other aetiologies. Neonatal
-pneumonia, however, is not included -- it is captured in our modelling of
+pneumonia, however, is not included -- it is captured in the modelling of
 pneumonia as a separate entity.
 
 .. todo::
-   Describe cause model
+   Add more information and references. In particular, find data about differences in global and 
+   gbd definitions, global prevalence and relation to folic acid during pregnancy.
+
+Modeling Measles in GBD 2017
+----------------------------
+
+It is worth noting that the ICD 10 and ICD 9 codes used in non-fatal and fatal GBD models are different. 
+The ICD 10 codes used in non fatal model are P36-P36.9, P38-P39.9. K29.0-K29.00, K29.1-K29.20, P77-P78.1 
+codes are also used in hospital/claims analyses of non fatal model The ICD 9 codes used are 771, 771.4-771.89. 
+For hospital/claims analyses, 535.0-535.00, 535.3-535.30, 777.5-777.7 are also used in addition to the above mentioned ICD 9 codes.
+While the fatal model uses only P36-P36.9, P38-P39.9 ICD 10 codes and 771.4-771.9 ICD 9 codes 
+[GBD-2017-YLD-Capstone-Appendix-1-Neonatal-sepsis-and-other-infections]_, [GBD-2017-CoD-Appendix-1-Neonatal-sepsis-and-other-infections]_.
+
```

Comment:
I think I am still confused about which codes were used where...

Reply:
Kiran and I had started to talk about this---but we should talk in coworking about whether these are important to list.

Reply:
Would it look better if I add a table like in YLD appendix but two rows one for fatal and the other for non-fatal?

Reply:
I have added a table for ICD codes. Hope it looks better now.